### PR TITLE
Adds extended Centcom support for ID console app and misc fixes

### PR DESCRIPTION
**Pull request by Timberpoes**
## About The Pull Request

Adds extended Centcom support including Centcom-class templates to Plexagon Access Management.

Patches out a few user input verification issues.

Fixes display issues including NaN/null issues with ID cards that have null ages.

image

## Why It's Good For The Game

Continued improvements. Missing functionality added.

## Changelog
:cl:
add: Centcom ID consoles have extended template support.
fix: ID cards with no registered age will no longer show null age in Plexagon Access Management.
/:cl:

### CODE REVIEW

File: code/modules/modular_computers/file_system/programs/card.dm [hunk 1]
```diff
@@ -200,14 +204,21 @@
 		if("PRG_age")
 			if(!computer || !authenticated_user || !target_id_card)
 				return TRUE
-			target_id_card.registered_age = params["id_age"]
+
+			var/new_age = params["id_age"]
+			if(!isnum(new_age))
+				stack_trace("[key_name(usr)] ([usr]) attempted to set invalid age \[[new_age]\] to [target_id_card]")
```
Comment (Mothblocks) [inline code/modules/modular_computers/file_system/programs/card.dm:210]: Do we need to worry about `html_encode` here or does the VV window do that for us?

File: tgui/packages/tgui/interfaces/NtosCard.js [hunk 2]
```diff
@@ -214,7 +214,7 @@ const IDCardTarget = (props, context) => {
             </Stack.Item>
             <Stack.Item>
               <NumberInput
-                value={id_age}
+                value={id_age ? id_age : 0}
```
Comment (Mothblocks) [inline tgui/packages/tgui/interfaces/NtosCard.js:217]:
```suggestion
                value={id_age || 0}
```

File: code/modules/modular_computers/file_system/programs/card.dm (hunk 1 above)
Comment (Timberpoes) [reply code/modules/modular_computers/file_system/programs/card.dm:210]: image

Is that what you were asking?

Comment (Mothblocks) [reply code/modules/modular_computers/file_system/programs/card.dm:210]: That works for me, yeah.

**Review by stylemistake — APPROVED** [review]

**Pull request merged**